index.bs

```diff
@@ -214,7 +224,7 @@ The <dfn>processorClose</dfn> algorithm is given a |processor| as input.
 It is defined by running the following steps:
 1. If |processor|.`[[isClosed]]` is true, abort these steps.
```

Comment:
Added Step 5. :)

Reply:
Ok, I think this is probably ok, since the readable stream queue should always be empty. If that were not the case, I think it would be possible for ReadableStreamClose to be run twice, which would result in an error.